pl_bolts/models/self_supervised/swav/swav_finetuner.py

```diff
@@ -0,0 +1,71 @@
+import os
+import pytorch_lightning as pl
+from argparse import ArgumentParser
+
+from pl_bolts.models.self_supervised.swav.swav_resnet import resnet50, resnet18
+
+from pl_bolts.models.self_supervised.ssl_finetuner import SSLFineTuner
+from pl_bolts.models.self_supervised.swav.swav_module import SwAV
+from pl_bolts.transforms.dataset_normalizations import stl10_normalization
```

Comment:
does this uses also `torchvision`?

Reply:
yes, but it is encapsulated in its own try catch